addons/lunch/views/lunch_orders_views.xml

```diff
@@ -216,57 +216,57 @@
                 <field name="available_today" invisible="1"/>
                 <div class="d-flex">
                     <div class="flex-grow-0 pe-5">
-                        <field name="image_1920" widget="image" class="o_lunch_image" options="{'image_preview': 'image_128'}"/>
+                        <field name="image_1920" widget="image" class="o_lunch_image" width="128" height="128" options="{'image_preview': 'image_128'}"/>
                     </div>
                     <div class="flex-grow-1 pe-5">
                         <h2><field name="name"/></h2>
                         <h3 class="pt-3"><field name="price"/></h3>
                     </div>
                 </div>
                 <div class="o_lunch_wizard">
-                    <div class="row">
-                        <div class="col-2">
+                    <div class="grid gap-1">
+                        <div class="g-col-12 g-col-md-2">
```

Comment:
Do not place labels on top, odoo's forms are always inline, in mobile too.

About the general layout, ideally this template should mirror the default `form-view` structure and classes, eg:
```xml

<table class="o_group o_inner_group">
    <tbody>
        <tr>
            <td class="o_td_label">
                <label class="o_form_label" for="o_field_input_60">Date</label>
            </td>
            <td style="width: 100%;">
                <span class="o_field_widget o_quick_editable" name="o_field_input_60">07/08/2022</span>
            </td>
        </tr>
    </tbody>
</table>
```
We'll probably switch to grid soon, I don't think it's worth to replicate the HTML structure, but we could use the same classes at least